src/connector/src/parser/config.rs

```diff
@@ -322,6 +322,7 @@ pub struct ProtobufProperties {
     pub topic: String,
     pub key_message_name: Option<String>,
     pub name_strategy: PbSchemaRegistryNameStrategy,
+    pub message_as_jsonb: Option<String>,
```

Comment:
Why not parse to `HashSet` here?

Reply:
I thought we'd accept a string here and parse it to HashSet when building a real protobuf parser.